src/Compilers/CSharp/Portable/BoundTree/BoundUnconvertedCollectionArguments.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal partial class BoundUnconvertedCollectionArguments
+    {
+        internal void GetArguments(AnalyzedArguments analyzedArguments)
```

Comment:
fwiw, havin thtis be called GetArguments and not return something reads oddly.  Consider .AddArgumentsTo, which makes it much clearer that the receiver is adding the arguments to the passed in value.

Reply:
I'd like to leave this as is. The intended use is to populate the `AnalyzedArguments` collection with just the arguments, rather than to add to an existing collection (we assert the incoming collection is empty initially), so renaming to `Add...` might make the intent less clear.

Reply:
Would "BuildArguments" or "CollectArguments" solve the problem?

Reply:
When we support collection builder types, where the factory method has an additional, implicit `ReadOnlySpan<T> elements` argument, it may make sense to rename this to `AddArguments()` or something similar. For now though, `GetArguments()` seems appropriate to me.